content/download/lts/index.adoc

```diff
@@ -92,8 +92,9 @@ See the link:/content/event-calendar[event calendar] for the specific LTS RC/rel
 
 ## Backporting Process
 
-Any user can propose that a issue is backported to LTS by labeling with link:https://issues.jenkins.io/secure/IssueNavigator.jspa?reset=true&jqlQuery=labels+%3D+lts-candidate[lts-candidate].
-For an issue to be eligible for backporting it should be a small, safe-looking, uncontroversial fix for an important bug with no change to API surface.
+Any user can propose that an issue is backported to LTS by labeling with link:https://issues.jenkins.io/secure/IssueNavigator.jspa?reset=true&jqlQuery=labels+%3D+lts-candidate[lts-candidate].
+For an issue to be eligible for backporting it should be a small, safe-looking, uncontroversial fix for an important bug with no change to API surface. +
+Backporting candidates have to be delivered in a weekly release to be tested by the community before shipping in LTS lines.
```

Comment:
Thanks.  I think the "should be" phrasing from the previous sentence should be applied here as well, since there may be an exceptional case where we would consider a backport without it being in a weekly release.
```suggestion
Backporting candidates should be delivered in a weekly release before shipping in an LTS release.
```